build.properties

```diff
@@ -10,10 +9,12 @@ mod_url=https://github.com/Team-RTG/Realistic-Terrain-Generation
 mod_author="Team RTG"
 mod_creds=Forked from RWG by ted80. OpenSimplex Noise by KdotJPG.
 mod_logo=assets/rtg/logo.png
-mcf_version=12.18.1.2011
-# mcf_postfix is the branch postfix (without '-') on the Forge version when it's not the default branch in the Forge repo
-# This should be same as mc_version, eg Non-default: 1.9.4-12.17.0.1974-1.9.4, Default: 1.9.4-12.17.0.1976
-mcf_postfix=
-mappings=snapshot_nodoc_20160724
+mcf_version=12.18.1.2094
```

Comment:
I've already been using this version of Forge and haven't noticed anything wrong. None of the changes made since 2011 have any effect on world gen that isn't backwards compatible. The min version is still set to 2011, so no one is forced to use the latest. The mappings don't make a difference as it only effects the deobf dev environment.